src/trade-params/TradeParams.js

```diff
@@ -85,6 +85,10 @@ export default class TradeParams extends Component {
         this.onStopTypeChange = ::this.onStopTypeChange;
         this.onStopProfitChange = ::this.onStopProfitChange;
         this.onPurchase = ::this.onPurchase;
+
+        this.state = {
+            dynamicKey: 0,
+        };
```

Comment:
@borisyankov alert , I am remounting the whole dom whenever asset change, this is to reset default value of inputs, and the reason of using default value is to allow user to input continuously.

I understand this is bad, so if you know any better way, please let me know, React is particularly bad in this aspect >.<